capitals.rb

```diff
@@ -150,3 +171,19 @@
     name: "Wyoming",
     capital: "Cheyenne"
 }]
+
+states = states.shuffle
+
```

Comment:
👍 you can also write
`states.shuffle!` 
the bang `!` returns a shuffled (modified) array